<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>465</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>295</width>
    <height>465</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>295</width>
    <height>465</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>D:/calculator.ico</normaloff>D:/calculator.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #000000;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #a6a6a6;
color: #000000;
font: 15pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>AC</string>
    </property>
   </widget>
   <widget class="QPushButton" name="invertButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #a6a6a6;
color: #000000;
font: 15pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>+/-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="divideButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff9601;
color: #ffffff;
font: 17pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="percentButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #a6a6a6;
color: #000000;
font: 15pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multiplyButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>170</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff9601;
color: #ffffff;
font: 17pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button8">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>170</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button7">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button9">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff9601;
color: #ffffff;
font: 17pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="subButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>230</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff9601;
color: #ffffff;
font: 17pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button6">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>230</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>350</y>
      <width>135</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dotButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>350</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #333333;
color: #ffffff;
font: 20pt &quot;Helvetica Light&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QPushButton" name="equalsButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>350</y>
      <width>54</width>
      <height>54</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ff9601;
color: #ffffff;
font: 17pt &quot;Helvetica&quot;;
text-align: center;
border-radius:27px;</string>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="inputField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica Light</family>
      <pointsize>30</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">color: #ffffff;
font: 30pt &quot;Helvetica Light&quot;;
border: 0px;
</string>
    </property>
    <property name="maxLength">
     <number>17</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>295</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
